wolfsentry/wolfsentry_errcodes.h

```diff
@@ -116,7 +116,7 @@ enum wolfsentry_source_id {
 };
 
 #ifdef WOLFSENTRY_ERROR_STRINGS
-WOLFSENTRY_API wolfsentry_errcode_t wolfsentry_user_source_string_set(enumint_t wolfsentry_source_id, const char *source_string);
+WOLFSENTRY_API wolfsentry_errcode_t wolfsentry_user_source_string_set(enum wolfsentry_source_id wolfsentry_source_id, const char *source_string);
```

Comment:
this looks like a reasonable change, but is it necessary?  are there knock-on effects?  I can't immediately recall why I had it as `enumint_t`...

Reply:
STM32 GCC errors without that change if `WOLFSENTRY_ERROR_STRINGS` is defined. Note that it is not defined by default in the Linux Makefile. I haven't tested changing that in Linux.